Makefile

```diff
@@ -52,3 +35,25 @@ docs-serve:
 .PHONY: docs-vale
 docs-vale:
 	$(VALE_CMD) $(VALE_ARGS)
+
+.PHONY: test-cloudscale
+test-cloudscale: testfile = cloudscale.yaml
+test-cloudscale: extra_args = -e CLOUDSCALE_TOKEN=sometoken
+test-cloudscale: .test
+
+.PHONY: test-exoscale
+test-exoscale: testfile = exoscale.yaml
+test-exoscale: extra_args = -e TF_VAR_lb_exoscale_api_key=ApiKeyForLoadBalancer -e TF_VAR_lb_exoscale_api_secret=ApiSecretForLoadBalancer
```

Comment:
setting `TF_VAR_lb_exoscale_api_key` somehow doesn't work when validating (both GH actions + locally). It could potentially also occur when actually running the GitLab CI pipelines...
Do you have an idea?

Reply:
I played around with it a bit just now and just realized that we need to introduce additional variables for the Ursula API key and secret in the `variables.tf.json` which is generated by the component, and pass those through to the actual Terraform module in order to be able to specify the key and secret as environment variables.

Reply:
I've got a commit locally which fixes the issue and simplifies input variable handling, but I'm not able to push it to the repo because I'm lacking permissions.

Reply:
Ups. That's fixed now